### ISSUE TITLE
Add download option to Control.Citation?

### ISSUE THREAD

**Issue by respinos** [mlibrary/cozy-sun-bear#131]
Based on the comments from the JPDF session, should we offer the ability to download the selected citation as well as copy to clipboard?

Possible formats: https://www.zotero.org/support/import_formats

**Comment by jmcglone** [comment]
Yes, especially because I think this would help move #49 (exporting citations - one of those formats is zotero) forward.

**Comment by mbakeryo** [comment]
We need to think through what we want to do before doing this.
